src/github/client/github-client-server.test.ts

```diff
@@ -4,11 +4,12 @@ import { GitHubInstallationClient } from "./github-installation-client";
 import { statsd }  from "config/statsd";
 import { InstallationId } from "./installation-id";
 import nock from "nock";
-import { AppTokenHolder } from "./app-token-holder";
-import fs from "fs";
-import { envVars }  from "config/env";
 import { GITHUB_ACCEPT_HEADER } from "utils/get-github-client-config";
+import { when } from "jest-when";
+import { booleanFlag, BooleanFlags } from "~/src/config/feature-flags";
+
 jest.mock("config/feature-flags");
+jest.mock("./key-locator");
```

Comment:
this is still being mocked in the test...

Reply:
Removed.